homeassistant/components/fritzbox_netmonitor/sensor.py

```diff
@@ -104,7 +103,6 @@ def state_attributes(self):
         attr = {
             ATTR_IS_LINKED: self._is_linked,
             ATTR_IS_CONNECTED: self._is_connected,
-            ATTR_WAN_ACCESS_TYPE: self._wan_access_type,
```

Comment:
Removing this state attribute is a breaking change. Please briefly describe the breaking change and what the user needs to do to cope with it, in the breaking change paragraph.

Reply:
Done, breaking change paragraph has been updated with this information.